### PR TITLE
Introduction of multiple energy ratings per asset

### PR DESCRIPTION
**Pull request by GokceBaykan**
This PR addresses the changes needed in the api-docs with regards to the introduction of multiple energy ratings per asset. Updates will span across: /energy-ratings.md, /assets.md examples, and /validation-endpoint.md

### CODE REVIEW

File: source/includes/_data-dictionary.md.erb
Comment (joeyhorst) [inline source/includes/_data-dictionary.md.erb:31]: would be the same as for `certifications`, namely `size`

it's okay to keep one row for both

just got to update the description: "Foor area covered by the certification or energy rating."

which has to be done in https://github.com/GRESB/api-docs/blob/master/data/dictionary.yml#L171

File: source/includes/_data-dictionary.md.erb
Comment (joeyhorst) [inline source/includes/_data-dictionary.md.erb:26]: this reminds me... 

certifications also have an additional identifier. This is meant to update/overwrite the certification record, in case you have more than one certification with the same brand/scheme, but a different level (gold and platinum for example)

now energy ratings do not have levels, but I do not know if we'll need an `id` to keep track of the energy rating record associated with the energy rating @bartj3? I would say this will not be the case, given there is no need to have two energy ratings with the same `energy_rating_id` associated with the same asset.

File: source/includes/_data-dictionary.md.erb
Comment (joeyhorst) [inline source/includes/_data-dictionary.md.erb:41]: description can be removed from https://github.com/GRESB/api-docs/blob/master/data/dictionary.yml#L168

File: source/includes/_energy-ratings.md
Comment (joeyhorst) [inline source/includes/_energy-ratings.md:34]: Good, but means we have to update it next year. I wonder if mentioning it in the version table is sufficient, and then we can keep this section generic? 

https://github.com/GRESB/api-docs/blob/master/data/versions.yml

same goes for the below 👇

File: source/includes/_data-dictionary.md.erb
Comment (joeyhorst) [reply source/includes/_data-dictionary.md.erb:26]: Thinking about it more, we need an `id` for energy ratings, otherwise how would you delete a energy rating from an asset? https://github.com/GRESB/api-docs/blob/master/source/includes/_certifications.md#remove-certifications

Setting values to `nill` would be a workaround, but let's align it with how certifications work to make it consistent.

File: source/includes/_energy-ratings.md
Comment (joeyhorst) [inline source/includes/_energy-ratings.md:38]: This was wrongly written to begin with and should follow the same structure as certifications.

This means that you make a normal POST or PATCH request (which we don't have to specify) with the required request json structure, now including `energy_ratings`.

https://github.com/GRESB/api-docs/blob/master/source/includes/_certifications.md#create-certifications

We might even want to combine building certifications and energy ratings at this stage - given they are almost identical (besides the `level` for certifications.

File: source/includes/_energy-ratings.md
Comment (joeyhorst) [inline source/includes/_energy-ratings.md:91]: depending on the `id` discussion, we do well also to add a response example

because the response will include `id` to be able to delete it afterwards

**Review by joeyhorst — CHANGES_REQUESTED** [review]
Looking at in detail it's unclear to me how we will allow API users to delete an energy rating. For which we need some input.

Either deletion is done by setting values to `nill` (rather not) or we offer an internal `id` to add a key `_destroy` (preferred).

At which stage it will be almost identical to certifications (which also has `level`) and we can combine the two sections.

File: source/includes/_energy-ratings.md
Comment (GokceBaykan) [reply source/includes/_energy-ratings.md]: I will create ticket to combine the two sections, I removed mention of PATCH method and made the text more generic for now.

File: data/dictionary.yml
Comment (00dav00) [inline data/dictionary.yml:170]:
```suggestion
  size: 'Floor area covered by the certification or energy rating.'
```

**Review by joeyhorst — APPROVED** [review]
looking good 👌 I'll have one final review of the section when we have all the changes in to check for consistency with the rest of the documentation
let's wait till we deployed the API changes before merging this, since it will automatically publish the new doc online with github actions

**Pull request merged**
